### ISSUE TITLE
it never loads up multiplayer as an option when i try to install

### ISSUE THREAD

**Issue by kiaboo3** [radistmorse/PEPatch#6]
idk if im installing it correctly or what but nothing works please help

**Comment by radistmorse** [comment]
idk all the more so if youre installing it correctly or what :) describe what you did, show some logs, something.

**Comment by kiaboo3** [comment]
i extract the pepatch master in planet explores game folder just the normal folder then i extract pepatch folder in the main folder and pe server patch in server then i configure my server

**Comment by kiaboo3** [comment]
i dont know if i should do something else or what

**Comment by radistmorse** [comment]
you probably are downloading the sources. You need the release zips. You can get them here https://github.com/radistmorse/PEPatch/releases/latest

**Comment by kiaboo3** [comment]
do i download the source zips

**Comment by kiaboo3** [comment]
or just the patches

**Comment by radistmorse** [comment]
No, you don't need the sources, just the patches. https://github.com/radistmorse/PEPatch/blob/master/README.md

**Comment by kiaboo3** [comment]
I managed to install it on my computer but my brothers computer wont install it and show to multiplayer option
